.github/workflows/ansible-test.yml

```
@@ -95,6 +95,9 @@ jobs:
         with:
           ansible-core-version: ${{ matrix.ansible }}
           testing-type: sanity
+          # Uncomment to configure a codecov token.
+          # Check here whether a token is required: https://docs.codecov.com/docs/codecov-tokens
```

I'd explicitly mention the ansible-collections namespace:
```suggestion
          # Check here whether a token is required: https://docs.codecov.com/docs/codecov-tokens
          # Collections in the github.com/ansible-collections/ namespace usually do not need a token.
```
Same for the others.

---

Makes sense! Fixed it :+1: